Cython/Utility/ModuleSetupCode.c

```diff
@@ -2230,3 +2230,21 @@ static PyObject* __Pyx_PyCode_New(
     Py_DECREF(varnames_tuple);
     return code_obj;
 }
+
+/////////////////////////// AccessPyMutexForFreeThreading.proto ////////////
+
+#if CYTHON_COMPILING_IN_CPYTHON_NOGIL
+// TODO - this is likely to get exposed properly at some point
+#ifndef Py_BUILD_CORE
+#define Py_BUILD_CORE 1
+#endif
+#include "internal/pycore_lock.h"
+#endif
+
+////////////////////////// SharedInFreeThreading.proto //////////////////
```

Comment:
Regarding naming, since we settled on `…_CPYTHON_NOGIL` for the target macro, I'd prefer sticking to that consistently. Throwing "free threading" into the mix just makes the context less clear.

Reply:
I've done a quick fixup https://github.com/cython/cython/pull/6210.